modules/side-panel/react/README.md

```diff
@@ -124,6 +124,18 @@ class SidePanelExample extends React.Component<{}, SidePanelState> {
 
 ---
 
+#### `hideNavigationLabel: string`
```

Comment:
Since we use the term `open`, can we change this to `closeNavigationLabel` and `openNavigationLabel`?

Reply:
that makes sense!!